averygenetti/splashzone-avery
# Customizable News Post Ads - committing solution

To test:
- Go to Django Admin Panel
- Go to Ads
- Click "Add Ad+" button
- Fill out form
- Create some non-house ads and at least one house ad (by checking 'house ad' box)
- Go to News section of admin panel
- Edit news posts and select various ads
- Observe that: news posts that have ads defined will show those ads when viewing the news post
- News posts that do not have ads defined will show a randomly selected ad that has 'house_ad' set to True
